Fix b64encode() for data starting with NUL from Miloslav Trmac <mitr@redhat.com>
--- beecrypt-4.1.2/base64.c.b64	2005-04-26 21:46:27.000000000 +0200
+++ beecrypt-4.1.2/base64.c	2005-04-26 21:47:48.000000000 +0200
@@ -253,7 +253,6 @@
     unsigned c;

     if (s == NULL)	return NULL;
-    if (*s == '\0')	return calloc(1, sizeof(*t));

     if (ns == 0) ns = strlen((const char*) s);
     nt = ((ns + 2) / 3) * 4;
